test/JDBC/input/BABEL_OBJECT_ID-vu-prepare.sql

```diff
@@ -0,0 +1,36 @@
+
+-- To test table, procedure, function
+CREATE TABLE babel_object_id_t1 (a int);
```

Comment:
Where are dependent objects on object_id?
We should add them as well for upgrade tests